lib/dir.js

```diff
@@ -240,17 +241,25 @@ function ensureDirectoryEmpty(req, res, next) {
 
 
 function deleteDirectory(req, res, next) {
+
+    assert.object(req, 'req');
+    assert.object(req.collector, 'req.collector');
+
     if (req.metadata.type !== 'directory')
         return (next());
 
     var client = req.moray;
+    var counter =
+        req.collector.getCollector(common.METRIC_DELETED_DIRECTORY_COUNTER);
     var log = req.log;
     var opts = {
         key: req.key,
         _etag: req.isConditional() ? req.metadata._etag : undefined,
         requestId: req.getId()
     };
 
+    assert.object(counter, 'METRIC_DELETED_OBJECT_COUNTER');
```

Comment:
Should this be 'METRIC_DELETED_DIRECTORY_COUNTER'?

Reply:
Yep, nice catch. Thanks!